notifier/selfstate/handle.go

```diff
@@ -0,0 +1,64 @@
+package selfstate
+
+import (
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/notifier/selfstate/checkable"
+)
+
+// CheckableHandle have target creation checkables data
+type CheckableHandle struct {
+	LastMetricReceivedTS, RedisLastCheckTS, LastCheckTS, LastRemoteCheckTS, NextSendErrorMessage *int64
+	MetricsCount, ChecksCount, RemoteChecksCount                                                 int64
+}
+
+// Init CheckableHandle base setting
+func (handler *CheckableHandle) Init(lastMetricReceivedTS, redisLastCheckTS, lastCheckTS, lastRemoteCheckTS, nextSendErrorMessage int64) {
+	handler.LastMetricReceivedTS = &lastMetricReceivedTS
+	handler.RedisLastCheckTS = &redisLastCheckTS
+	handler.LastCheckTS = &lastCheckTS
+	handler.LastRemoteCheckTS = &lastRemoteCheckTS
+	handler.NextSendErrorMessage = &nextSendErrorMessage
+}
+
+// Create array structures Checkable
+func (handler CheckableHandle) Create(log moira.Logger, db moira.Database, config Config) []checkable.Checkable {
+	if log == nil || db == nil {
```

Comment:
`log` and `db` are not pointers so checking for nillness is not correct

Reply:
it is interface